basyx.submodelservice/basyx.submodelservice.component/example/Readme.md

```diff
@@ -0,0 +1,19 @@
+## Submodel Service Component Example
+
+This example describes a potential setup for the Submodel Service using Docker Compose.
+
+## Configuration
+
+The [docker-compose.yml](docker-compose.yml) file provides a basic setup for starting the service. The Docker image used is initially built based on the [Dockerfile located in the parent directory](../Dockerfile).
+
+Volumes are used to provide the Submodel and the executable source code to the container, which are referenced in the environment section. The mapping of `idShortPath` to Java classes is also referenced there and loaded via [application-mappings.yml](application-mappings.yml). Alternatively, you can simplify the setup by configuring everything directly in [application.yml](application.yml).
```

Comment:
I found this paragraph a bit confusing. The example/application.yml is not being used by the example/docker-compose.yml or in any of the integration tests. I understand it's there to serve as an example for a full configuration.

To improve understandability and clarity, the standalone example could be moved to the ./examples folder (root dir). I know some of the files are in here are being used for the integration tests; consider refactoring the package such, that the integration files are separated from the example itself and the full `application.yml` is used in the example.